addon/components/detail.js

```diff
@@ -157,9 +163,10 @@ export default Component.extend(PropTypeMixin, {
   init () {
     this._super()
 
-    const reduxStore = createStoreWithMiddleware(reducer)
+    const reduxStore = createStoreWithMiddleware(reducer, initialStore({baseModel: this.get('model')}))
 
     this.set('reduxStore', reduxStore)
+    this.set('reduxModel', reduxStore.getState().model)
```

Comment:
I guess we need to capture this property along with the store itself so we can track changes.